testpilot/frontend/static-src/app/views/testpilot-promo-view.js

```diff
@@ -0,0 +1,56 @@
+import BaseView from './base-view';
+import installAddon from '../lib/install-addon';
+
+
+export default BaseView.extend({
+  _template: `
+    <div class="experiment-promo">
+      <div class="reverse-split-banner responsive-git content-wrapper">
+        <div class="copter-wrapper fly-up">
+          <div class="copter"></div>
+        </div>
+        <div class="intro-text">
+          <h2 class="banner">
+            <span data-l10n-id="experimentPromoHeader" class="block">Ready for Takeoff?</span>
+          </h2>
+          <p data-l10n-id="experimentPromoSubheader">We're building next-generation features for Firefox. Install Test Pilot to try them!</p>
+          {{^isFirefox}}
+            <span data-l10n-id="landingDownloadFirefoxDesc" class="parens">(Test Pilot is available for Firefox on Windows, OS X and Linux)</span>
+            <a href="https://www.mozilla.org/firefox" class="button primary download-firefox">
+              <div class="button-icon">
+                <div class="button-icon-badge"></div>
+              </div>
+              <div class="button-copy">
+                <div data-l10n-id="landingDownloadFirefoxTitle" class="button-title">Firefox</div>
+                <div data-l10n-id="landingDownloadFirefoxSubTitle" class="button-description">Free Download</div>
+              </div>
+            </a>
+          {{/isFirefox}}
+          {{#isFirefox}}
+          <button data-hook="install" class="button extra-large primary">
```

Comment:
Since we added another install button, we should add an integration test for it. Add-on installation flow is one of our sensitive spots that's ended up broken a lot in the past month or two. Don't want to block this PR, though, since it's been enough of a journey so far. Filed a follow up issue: https://github.com/mozilla/testpilot/issues/1257